docs/Externalization.md

```diff
@@ -0,0 +1,32 @@
+# Storage Externalization
+
+## --link
```

Comment:
Make the structure clearer that the first half of the doc is for the user and the second half is for the administrator.  Some more context and motivation is probably required to fully appreciate this - not for the public instance, but for private instances of CodaLab on a shared file system...and this allows you to mutate the underlying files (useful for debugging code).

Reply:
Updated. To be clear in the documentation, I said that `--link` is used for "private instances of CodaLab in which the server and users both have access to the same filesystem", because the usage of `--link` is orthogonal to the `--shared-file-system` tag (which refers to the server and workers sharing a filesystem).